common/changes/office-ui-fabric-react/u-mahuangh-fix-onSuggestionRemove-types_2019-02-08-00-41.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Re-type onItemRemove in Suggestions to support the suggestions Generic type",
```

nit: 😄  for release note scanability))
```suggestion
      "comment": "Pickers: re-type onItemRemove in Suggestions to support the suggestions Generic type",
```